### ISSUE TITLE
Continue.Dev into Ace -> https://ace.c9.io/

### ISSUE THREAD

**Issue by supercharles76** [continuedev/continue#477]
**Is your feature request related to a problem? Please describe.**
Just imagine the same (wonderful) integration inside ace editor. 

**Describe the solution you'd like**
You may integrate a sidepanel near to the Ace Editor.

**Additional context**
<img width="901" alt="raw_continue_ace" src="https://github.com/continuedev/continue/assets/74455199/b5cec286-f80b-48c5-8854-19f7a2affced">

**Comment by sestinj** [comment]
Loving the idea! We are just today releasing the (very much in alpha) version a JetBrains extension, so Ace is also possible.

I do want to focus first on building a super high quality version for JetBrains before expanding to others, but if you wanted to try yourself in the meantime, it is actually fairly straightforward to get the basic features—mostly just need to place the window at the side. Is building an Ace extension something you've done before by chance?

**Comment by supercharles76** [comment]
No, I've never done that before. I'm just a graphic web designer
